Calibration/LumiAlCaRecoProducers/plugins/RawPCCProducer.cc

```diff
@@ -95,37 +97,36 @@ void RawPCCProducer::globalEndLuminosityBlockProduce(edm::LuminosityBlock& lumiS
   //new vector containing clusters per bxid with afterglow corrections
   std::vector<float> corrClustersPerBXOutput(LumiConstants::numBX, 0);
 
-  //The indicies of all the good modules - not vetoed
-  std::vector<int> goodMods;
-
+  //////////////////////////////////
+  /// read input , clusters per module per bx
+  /////////////////////////////////
   edm::Handle<reco::PixelClusterCounts> pccHandle;
   lumiSeg.getByToken(pccToken_, pccHandle);
-
   const reco::PixelClusterCounts& inputPcc = *(pccHandle.product());
```

Comment:
The handle is not used directly, so would make sense to omit it. I think there should be a method to get the product directly